src/style.css

```diff
@@ -1,3 +1,74 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: none;
+  list-style: none;
+  box-sizing: border-box;
```

Comment:
Avoid using the '*' selector as it impacts performance. It's better to set styles only for elements that require them.